ample/util/argparse_util.py

```diff
@@ -89,7 +89,7 @@ def add_general_options(parser=None):
     parser.add_argument('-nmr_process', type=int, help='number of times to process the NMR models')
     parser.add_argument('-nmr_remodel', metavar='True/False', help='Remodel the NMR structures')
     parser.add_argument('-nmr_remodel_fasta', help='The FASTA sequence to be used for remodelling the NMR ensemble if different from the default FASTA sequence')
-    parser.add_argument('-purge', metavar='True/False', help='Delete all intermediate files and failed MRBUMP results')
+    parser.add_argument('-purge', metavar='purge_level', help='Delete all intermediate files and failed MRBUMP results')
```

Comment:
Could you provide more information about the possible purge levels, perhaps you can add in `choices=[0, 1]` to show the different purge_levels. 

Alternatively if you are just replacing True/False with 1/0 in order to get around the awkward way that argparse handles input bool arguments you could do something like this: 

`parser.add_argument('--purge', default=False, action="store_true", help="Delete all intermediate files and failed MRBUMP results")`

Where supplying `--purge` to the command line would return True

Reply:
I've just see that there are multiple clean up options, so ignore the latter part, just make the options more clear